src/components/ResultBar/ResultSort/ResultSort.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classes from './ResultSort.less';
+
+const resultsort = (props) => {
+	const sortby = props.sorttypes.map((sort, index) => {
+		let sortbyClass = classes.sortby;
+		if(sort.selected){
+			sortbyClass = [classes.sortby, classes.selected].join(' ');
+		}
+		return <p className={sortbyClass} selected={sort.selected} onClick={props.click} key={index}>{sort.title}</p>
```

Comment:
already wrote previously: 

Props for passing handlers better to name starting with `-on` prefix. As an example in react: onClick, onBlur, etc.
Handlers methods better to name starting with `handle-` prefix.